docs/docsite/rst/dev_guide/testing_httptester.rst

```diff
@@ -0,0 +1,71 @@
+**********
+httptester
+**********
+
+.. contents:: Topics
+
+Overview
+========
+
+``httptester`` is a docker container used to host certain resources required by :doc:`testing_integration`, this is to avoid CI tests requiring external resources, such as git or package repos, which if temporarily unavailable would cause tests to fail.
+
+HTTP Testing endpoint which provides the following capabilities:
+* httpbin
```

Comment:
Looks like these are bullet points. There must be an empty line right after the sentence in order to have the bullets.